libavfilter/dnn/dnn_backend_tf.c

```diff
@@ -600,6 +603,7 @@ static DNNReturnType load_native_model(TFModel *tf_model, const char *model_file
 
 #define CLEANUP_ON_ERROR(tf_model) \
     { \
+        TF_DeleteTensor(tensor); \
```

Comment:
add TF_DeleteTensor here, as I said TF_DeleteXXX(NULL) will not report error, which means it's ok to call CLEAN_UP_ON_ERROR in line 618 even if 'tensor' is not allocated.